### DISCUSSION TITLE
AutoEncoders in Jax, Flax

### DISCUSSION THREAD

**Discussion by nipunbatra** [google/flax#2602] [Show and tell]
Here's a post I've written (and will continue expanding) on auto encoders in Flax.

https://nipunbatra.github.io/blog/posts/autoencoder.html

Any inputs/suggestions/ideas/contributions would be most welcome! 

My aim is to be able to create an educative blog similar in model coverage to the [Keras blog post on auto encoders](https://blog.keras.io/building-autoencoders-in-keras.html).

**Comment by cgarciae** [comment]
Amazing @nipunbatra! Happy to RT if when you publish it :)
